src/Database/Esqueleto/Experimental.hs

```diff
@@ -767,6 +771,7 @@ instance ( ToAlias a
 type family ToAliasReferenceT a where
   ToAliasReferenceT (SqlExpr (Value a)) = SqlExpr (Value a)
   ToAliasReferenceT (SqlExpr (Entity a)) = SqlExpr (Entity a)
+  ToAliasReferenceT (SqlExpr (Maybe (Entity a))) = SqlExpr (Maybe (Entity a))
```

Comment:
This is a very unrelated question, but why do both `ToAliasT` and `ToAliasReferenceT` exist? They seem to do the same thing (?)

Reply:
They might be able to be merged but it's not clear to me that they will always be the same. They initially started as associated type families in their respective classes. Semantically they are different though, an alias is something like "value as alias" where an alias reference is like "q.v".